hazelcast/src/main/java/com/hazelcast/multimap/impl/MultiMapDataSerializerHook.java

```diff
@@ -304,6 +307,12 @@ public IdentifiedDataSerializable createNew(Integer arg) {
                 return new EntrySetResponse();
             }
         };
+        constructors[DELETE] = new ConstructorFunction<Integer, IdentifiedDataSerializable>() {
+            @Override
+            public IdentifiedDataSerializable createNew(Integer arg) {
+                return new DeleteOperation();
+            }
+        };
```

Comment:
The `DELETE_BACKUP` constructor is missing.

Reply:
Thanks, fixed.